### PR TITLE
Not found page

### PR DESCRIPTION
**Pull request by Pablo-Mdz**
## Description

I built the page not found using figma, but the letters look a bit big to me, please check it and if you like, approve the PR

- [x] New feature (non-breaking change which adds functionality)

## Screenshots 

Screenshot 2023-04-17 at 17 19 52

### CODE REVIEW

**Review by antoniosaleme — APPROVED** [review]
LGTM!

File: client/src/pages/NotFoundPage/NotFoundPage.jsx [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+        <p className="text-3xl text-gray-700 mb-6 mx-10">
+          Error 404. Like falling off a seesaw, things don’t go as planned and
+          we can’t you connect to this page.
+        </p>
+        <a
+          href="/"
+          className="inline-block bg-purple-400  rounded-full px-3 py-1 text-xl font-semibold text-white hover:bg-gray-700 mr-2 mb-2  hover:text-white"
+        >
+          Return Home
+        </a>
```
Comment (antoniosaleme) [inline client/src/pages/NotFoundPage/NotFoundPage.jsx]: this one should be 266px x 31px(better use 32px)

File: client/src/pages/NotFoundPage/NotFoundPage.jsx [hunk 2]
```diff
@@ -1,8 +1,27 @@
...
-    <div>
-      <h1>Page Not Found</h1>
-      <p>This page doesn't seem to exist</p>
+    <div className="h-screen flex font-pop">
+      <div className="w-1/4 bg-white flex flex-col justify-center items-center p-8 my-10">
+        <h1 className="text-6xl font-bold text-gray-900 mb-4">Uh-oh!</h1>
+        <p className="text-3xl text-gray-700 mb-6 mx-10">
+          Error 404. Like falling off a seesaw, things don’t go as planned and
+          we can’t you connect to this page.
+        </p>
```
Comment (antoniosaleme) [inline client/src/pages/NotFoundPage/NotFoundPage.jsx]: text should be text-4xl 
margin bottom should be mb-9 
line-hight should be leading-9

**Review by antoniosaleme — CHANGES_REQUESTED** [review]

File: client/src/pages/NotFoundPage/NotFoundPage.jsx (hunk 2 above)
Comment (Pablo-Mdz) [reply client/src/pages/NotFoundPage/NotFoundPage.jsx]: with leading-10 is a bit better I think.

**Pull request merged**